src/peer-connection.ts

```diff
@@ -285,6 +285,45 @@ class PeerConnection extends EventEmitter<PeerConnectionEventHandlers> {
   get iceGatheringState(): RTCIceGathererState {
     return this.pc.iceGatheringState;
   }
+
+  /**
+   * Returns the type of a connection that has been established.
+   *
+   * @returns The connection type which would be {Promise<'UDP' | 'TCP' | 'TURN-TLS' | 'TURN-TCP' | 'TURN-UDP' | 'unknown'>}.
+   */
+  async getCurrentConnectionType(): Promise<string> {
```

Comment:
Wondering if it's worth explicitly defining a type for these: instead of `string`, doing something like:

```
const ConnectionType = 'UDP' | 'TCP' | 'TURN-TLS' | 'TURN-TCP' | 'TURN-UDP' | 'unknown';
```

Reply:
Good suggestion, I have already updated this code .